scipy-1.0/paper.tex

```diff
@@ -83,23 +83,26 @@ \subsection*{Polynomial interpolators}
 \subsection*{Test and benchmark suite}
 
     \subsubsection*{Benchmark suite}
+
     The airspeed velocity (asv) library enables benchmarking Python packages over their lifetimes, and the performance of the SciPy
     code base was monitored with asv starting in February of 2015 (PR \#4501). In addition to ensuring that unit tests are passing (see above),
     confirming that performance generally remains constant or improves over the commit hash history of the project allows us to objectively
     measure that our code base is improving, to empower scientific applications.
 
-    Consider the following asv benchmark results, spanning roughly nine years of project history, that demonstrate the gradual performance
-    improvements in \texttt{scipy.spatial.cKDTree.query()} (nearest-neighbor search) performance, using command:
+    Consider the asv benchmark results shown in Figure~\ref{fig:asvbench}, spanning roughly nine years of project history. These demonstrate the gradual performance
+    improvements in a nearest-neighbor search through \texttt{scipy.spatial.cKDTree.query()}, and can be run using the command:
+
 
     \texttt{python run.py run -e -s 800 --bench "\textbackslash btime\_query\textbackslash b" "02de46a546..b3ddb2c"}
 
-    \begin{figure}[H]
+    \begin{figure}
```

Comment:
Removing the locator flag here lands Figure 1 in the numerical optimization section.